### ISSUE TITLE
Color calibration instance after import shows adaptation as none bypass

### ISSUE THREAD

**Issue by chhil** [darktable-org/darktable#7211]
Current master.

Imported a folder with fuji raw+jpeg.

Color calibration shows up with adaptation as none/bypass on one image, is this normal?
![image](https://user-images.githubusercontent.com/1119439/101271516-73660a80-37a9-11eb-8ae2-608040940bb8.png)

I have never seen this till date.

When I hit reset on it it changes to 
![image](https://user-images.githubusercontent.com/1119439/101271572-00a95f00-37aa-11eb-83d5-aa4a799894a0.png)

What should reset set it to, what are the defaults?

**Comment by chhil** [comment]
When I change the cct to linear bradford
![image](https://user-images.githubusercontent.com/1119439/101271600-57af3400-37aa-11eb-85b2-22de945c7569.png)


Then hit the cct color picker

![image](https://user-images.githubusercontent.com/1119439/101271618-6eee2180-37aa-11eb-9844-4b7d0db9efc1.png)

It changes adaptation to CAT16. Should adaptation change when cct color picker is used?

**Comment by Nilvus** [comment]
Bypass depend on setting "auto-apply chromatic adaptation by default" in processing tab in preferences. If white balance is used by legacy module white balance, this should be in bypass mode to not apply twice a white balance color. If balance color is set to D65 (bulb light icon) in white balance, then you could use CAT16 or Bradford. That last way is applied automatically if setting cited is set to "modern".

Color picker detect colors AND which adaptation seems better to be used on the image, so yes it's normal that you could see on some images a change from Bradfort to CAT16 or other way.

All this is explained and more on Aurélien Pierre article in pixl.us: https://discuss.pixls.us/t/introducing-color-calibration-module-formerly-known-as-channel-mixer-rgb/21227

You sure will learn some things about that new great module.

**Comment by chhil** [comment]
I have "modern" setup in preferences
White balance has the bulb selected.
Hence did not expect  bypass none to be selected in the color calibration.

This stood out for one image only in the 100+ that I imported.

![image](https://user-images.githubusercontent.com/1119439/101280893-d4193580-37f1-11eb-99f2-fe10076e83d9.png)

**Comment by Nilvus** [comment]
If it's stood on one image, it will be difficult to find why. Maybe setting was not set when that image was imported.

**Comment by Nilvus** [comment]
Or an existing preset that override it for some images? Just an idea!

**Comment by chhil** [comment]
No presets. 
Setting of modern always on. 
Thus image was in the middle of all the folder images imported.

**Comment by Nilvus** [comment]
So, quite strange! And seems hard to reproduce, so to solve by some developer. But maybe someone else will have a better idea there.

**Comment by TurboGit** [comment]
Does this happen if you import only RAW and not the jpeg ? Can you try on a directory without jpeg ? This may give an idea about the issue at hand. TIA.

**Comment by AlicVB** [comment]
and maybe just a quick test : if you do a copy of the problematic file with system tools (without the xmp) and import this copy, does the problem appears again ? If yes, that can be an indication that there's a problem with this specific file, so maybe you can post the raw somewhere...

**Comment by chhil** [comment]
**Test 1**
I removed all raf+jpg from the collection. 
Deleted all xmps.
Imported just the **single** RAF.
I dont see the problem, though I could swear the last time it imported as a portrait and this time its a landscape.

![image](https://user-images.githubusercontent.com/1119439/101298076-43267680-3852-11eb-984e-0d3e832a010e.png)

**Test 2**
Removed files from collection.
Deleted all xmps
I import the **single jpg + raw.

**
**The jpeg is portrait but raf is landscape.** Color calib is correctly not bypass.

**Test 3**
Removed all images from collection
Removed all xmps.
Imported just RAF files.
Color calib was fine, image was correctly portrait.

**Test 4**
Removed all images from collection
Removed all xmps.
Imported all RAF+jpeg files.
Color calib was fine, image was correctly portrait.


Bottom line could not reproduce the issue but see a new issue with orientation of RAF file.
The issue I reported did happen, unfortunately I cannot reproduce it.

jpeg and raf file can be found here
https://www.dropbox.com/sh/drfxop19ry0jc2h/AAAPoKrL0zO4E_l9wECQWcaNa?dl=0

**Comment by Mark-64** [comment]
Can reproduce 100% on both Windows and Ubuntu 20.04 with latest git.
Steps to reproduce:
1) start DT with an empty database
2) import a folder with only RAW files, no xmp (I used a folder with Canon, Nikon, Sony, Fuji and Pentax files)
3) Open one random file in darkroom, see CAT is applied
4) While in darkroom, switch to other images with filmstrip. CAT is not applied, set to none (bypass)

**Comment by Nilvus** [comment]
> Can reproduce 100% on both Windows and Ubuntu 20.04 with latest git.
> Steps to reproduce:
> 
>     1. start DT with an empty database
> 
>     2. import a folder with only RAW files, no xmp (I used a folder with Canon, Nikon, Sony, Fuji and Pentax files)
> 
>     3. Open one random file in darkroom, see CAT is applied
> 
>     4. While in darkroom, switch to other images with filmstrip. CAT is not applied, set to none (bypass)

But with which chromatic adaptation setting in processing tab in preferences window? If it's legacy, it's a normal behavior.

**Comment by Mark-64** [comment]
> 
> 
> > Can reproduce 100% on both Windows and Ubuntu 20.04 with latest git.
> > Steps to reproduce:
> > ```
> > 1. start DT with an empty database
> > 
> > 2. import a folder with only RAW files, no xmp (I used a folder with Canon, Nikon, Sony, Fuji and Pentax files)
> > 
> > 3. Open one random file in darkroom, see CAT is applied
> > 
> > 4. While in darkroom, switch to other images with filmstrip. CAT is not applied, set to none (bypass)
> > ```
> 
> But with which chromatic adaptation setting in processing tab in preferences window? If it's legacy, it's a normal behavior.

Yes, chromatic adaptation set to "modern" in preferences

**Comment by Mark-64** [comment]
Solved for me with the latest commits in master

**Comment by TurboGit** [comment]
So closing.
